app/actions/WalletLoaderActions.js

```diff
@@ -53,11 +54,27 @@ export const CREATEWALLET_NEWSEED_CONFIRM_INPUT = "CREATEWALLET_NEWSEED_CONFIRM_
 export const CREATEWALLET_NEWSEED_BACK_INPUT = "CREATEWALLET_NEWSEED_BACK_INPUT";
 export const CREATEWALLET_EXISTINGSEED_INPUT = "CREATEWALLET_EXISTINGSEED_INPUT";
 export const CREATEWALLET_GOBACK_EXISITNG_OR_NEW = "CREATEWALLET_GOBACK_EXISITNG_OR_NEW";
+export const CREATEWALLET_GOBACK = "CREATEWALLET_GOBACK";
 export const CREATEWALLET_NEWSEED_INPUT = "CREATEWALLET_NEWSEED_INPUT";
 
 export const createWalletConfirmNewSeed = () => ({ type: CREATEWALLET_NEWSEED_CONFIRM_INPUT });
 export const createWalletGoBackNewSeed = () => ({ type: CREATEWALLET_NEWSEED_BACK_INPUT });
 export const createWalletGoBackExistingOrNew = () => ({ type: CREATEWALLET_GOBACK_EXISITNG_OR_NEW });
+
+export const createWalletGoBackWalletSelection = () => (dispatch, getState) => {
+  const { daemon: { walletName, network } } = getState();
+  wallet.stopWallet().then(() => {
+    wallet.removeWallet(walletName, network == "testnet")
+      .then(() => {
+        dispatch({ type: CREATEWALLET_GOBACK });
+        dispatch(getAvailableWallets());
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
Probably wanna dispatch a `{error, type: REMOVEWALLET_FAILED}` or similar here, as otherwise in case of an error the user might reach the 3 wallet limits without having any feedback an error has happened.

And then add the action type on the snackbar reducers here: https://github.com/decred/decrediton/blob/master/app/reducers/snackbar.js#L240